ci/azure-cleanup/purge_old_agents.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i python3 .
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script purges agents older than 25 hours in a given pool
+# Pass PAT and POOL_ID as environment variables to it
+import datetime
+import os
+import sys
+
+from azure.devops.connection import Connection
+from msrest.authentication import BasicAuthentication
+
+ORGANIZATION_URL = "https://dev.azure.com/digitalasset"
+PAT = os.environ["PAT"]
+POOL_ID = os.environ["POOL_ID"]
+
+credentials = BasicAuthentication('', PAT)
+connection = Connection(base_url=ORGANIZATION_URL, creds=credentials)
+
+agent_client = connection.clients_v5_1.get_task_agent_client()
+
+for agent in agent_client.get_agents(POOL_ID):
+    # agents should be killed after 24 hours max
+    cutoff_time = datetime.datetime.now(datetime.timezone.utc)\
+                  - datetime.timedelta(hours=25)
```

Comment:
can you move this into a constant to give it a bit more visibility?

Reply:
fixed.